test/transport.browser.spec.ts

```diff
@@ -89,3 +94,28 @@ describe('basic transport tests', () => {
   });
 });
 
+import { SERVER_MULTIADDR } from './server-multiaddr';
+
+describe('Transport interoperability tests', () => {
+  it('can connect to a server', async () => {
+    if (SERVER_MULTIADDR) {
+      console.log('Will test connecting to', SERVER_MULTIADDR);
+    } else {
+      console.log('Will not test connecting to an external server, as we do not appear to have one.');
```

Comment:
With `export var SERVER_MULTIADDR = '';`, does this test never run?

Reply:
It gets rewritten through an npm script. cc @John-LittleBearLabs

Reply:
Yeah. If you're doing test and not test:interop then it doesn't get overwritten and you won't run this.